mlir/include/air/Transform/Passes.td

```diff
@@ -1159,6 +1159,10 @@ def AIRLabelBroadcastChannelWithTilePass : Pass<"air-label-broadcast-channel-wit
 def AIRCollapseHerdPass : Pass<"air-collapse-herd", "func::FuncOp"> {
   let summary = "Collapse a multi-dimensional air.herd into a single column.";
   let constructor = "xilinx::air::createAIRCollapseHerdPass()";
+  let options = [
+    Option<"clMaxColSize", "max-col-size", "int", /*default=*/"4",
+           "The maximum column size after collapse, before collapse is cancelled.">
+  ];
```

Comment:
I think this is a good idea. The reason why 4 as default is convenient is because IREE-AMD-AIE wouldn't need to update its pass registration.

Let me not merge this PR for now, until I put together a PR at IREE-AMD-AIE where I update their pass registration, too.

Reply:
Done.